README.md

```diff
@@ -6,7 +6,22 @@ Erudio HTTP library to fetch data from various endpoint/GraphQL
 
 ```
 const erudioClient = new ErudioClient(<domain>);
+
+// Get 10 structure nodes for given namespace
 const nodes = erudioClient.getStructures( 'f07e196f-e4bd-46f0-97a5-c3ae768683dc', {limit:10});
+
+// Get single Node filled with data, for given structureId
 const singleNode = erudioClient.getStructureNode( 'f07e796f-e3bd-46f0-97a5-c3ae7409dc', 'en');
-const tags = erudioClient.getTags(ServiceType.CONTENT, 'f07e796f-e3bd-46f0-97a5-c3ae7409da')
+
+// Get all the tags for given object type and object id.
+const tags = erudioClient.getTags(ServiceType.CONTENT, 'f07e796f-e3bd-46f0-97a5-c3ae7409da');
+
+// Get all the links by given namespaceId
+const links = erudioClient.getStructureLinks({targetNamespaceId: 'f07e796f-e3bd-46f0-97a5-c3ae7409dc'});
```

Comment:
@matuz @iapain we should have some consistency in the API: `getStructures` have `namespaceId` as its first parameter, then `options`, here we have an object with `targetNamespaceId` key (which IMO should be rather `namespace` or `namespaceId` if we decide to go with passing an object here)

Reply:
ok, now I see it's becuase of the way params are defined in the structure link service